lab8-sealed-class.md

```diff
@@ -199,7 +212,7 @@ final public class Lab extends Breakout {
 🔎 Both classes are `final`.
 
 
-4. Create a fictional `AgendaRepository.java` class
+4. Create a final `AgendaRepository.java` class
```

Comment:
I really meant fictional here ;-)

Reply:
I'm kinda confused why the word "fictional" is here then. "fictional" here would imply that this is a mental exercise, rather than actual steps an attendee would go through.